app/src/main/java/com/example/d2android100/data/Mapper.kt

```diff
@@ -3,28 +3,28 @@ package com.example.d2android100.data
 import com.example.d2android100.domain.ShopItem
 
 class Mapper {
-    fun mapEntityToDbModel(shopItem : ShopItem) : ShopItemDbModel {
-        return ShopItemDbModel(
-            id = shopItem._id,
-            name = shopItem.name,
-            count = shopItem.count,
-            enabled = shopItem.enabled
-        )
-    }
-    fun mapDbModelToEntity(shopItemDbModel : ShopItemDbModel) : ShopItem{
-        return ShopItem(
-            _id = shopItemDbModel.id,
-            name = shopItemDbModel.name,
-            count = shopItemDbModel.count,
-            enabled = shopItemDbModel.enabled
-        )
-    }
-
-    fun mapListDbModelToListEntity(list : List<ShopItemDbModel>) : List<ShopItem>{
-        val shopItemList = mutableListOf<ShopItem>()
-        for (shopItem in list){
-            shopItemList += (mapDbModelToEntity(shopItem))
-        }
-        return shopItemList
-    }
+//    fun mapEntityToDbModel(shopItem : ShopItem) : ShopItemDbModel {
+//        return ShopItemDbModel(
+//            id = shopItem._id,
+//            name = shopItem.name,
+//            count = shopItem.count,
+//            enabled = shopItem.enabled
+//        )
+//    }
+//    fun mapDbModelToEntity(shopItemDbModel : ShopItemDbModel) : ShopItem{
+//        return ShopItem(
```

Comment:
` for (shopItem in list){
            shopItemList += (mapDbModelToEntity(shopItem))
        }`